app/lib/common/dialogs/attachment_selection.dart

```diff
@@ -0,0 +1,104 @@
+import 'dart:io';
+
+import 'package:acter/common/dialogs/attachment_confirmation.dart';
+import 'package:acter/common/utils/utils.dart';
+import 'package:acter/common/widgets/attachments/attachment_options.dart';
+import 'package:acter_flutter_sdk/acter_flutter_sdk_ffi.dart'
+    show AttachmentsManager;
+import 'package:file_picker/file_picker.dart';
+import 'package:flutter/material.dart';
+import 'package:image_picker/image_picker.dart';
+
+// reusable dialog for attachment selection
+void showAttachmentSelection(
+  BuildContext context,
+  AttachmentsManager manager,
+) async {
+  isLargeScreen(context)
```

Comment:
This can be regarded as something outside of this PR scope. I would have open it as separate ticket if there's a general take. Besides that, desktop attachments design are still unclear as Balsmiq references mobile view.